sdformat_to_mjcf/sdformat_to_mjcf/sdf_kinematics.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sdformat as sdf
+
+
+class LinkNode:
```

Comment:
This hasn't been merged yet. I think we should limit this PR to files that have been reviewed and merged.